src/components/StatusDashboard/current_migrations.jsx

```diff
@@ -0,0 +1,268 @@
+              select={() => select("longterm")}
+              sort={state.sort}
+            />
+            <TableContent
+              collapsed={state.collapsed.regular}
+              name="Regular migrations"
+              resort={resort}
+              rows={regular}
+              select={() => select("regular")}
+              sort={state.sort}
+            />
+            <TableContent
+              collapsed={state.collapsed.closed}
+              name="Closed migrations"
+              resort={resort}
+              rows={closed}
+              select={() => select("closed")}
+              sort={state.sort}
+            />
+          </table>
+        </div>
+      </div>
+    </>
+  );
+}
+
+function TableContent({ collapsed, name, resort, rows, select, sort }) {
+  const [redirect, setState] = useState('');
+  if (redirect) return (<Redirect to={redirect} />);
+  return (
+    <>
+      <thead>
+        <tr onClick={select}>
+          <th colSpan="7" className={collapsed ? styles.collapsed : undefined}>
+            {name}{" "}
+            <span className="badge badge--secondary">{rows.length || "…"}</span>
+          </th>
+        </tr>
+        <tr className={collapsed ? styles.collapsed : undefined}>
+          <th
+            onClick={() => resort("name")}
+            className={sort.by === "name" ? styles[sort.order] : undefined}
+          >
+            Name
+          </th>
+          <th
+            onClick={() => resort("status")}
+            className={sort.by === "status" ? styles[sort.order] : undefined}
+          >
+            Status
+          </th>
+          <th
+            onClick={() => resort("awaiting-parents")}
+            className={sort.by === "awaiting-parents" ? styles[sort.order] : undefined}
+          >
+            Awaiting parents
+          </th>
+          <th
+            onClick={() => resort("awaiting-pr")}
+            className={sort.by === "awaiting-pr" ? styles[sort.order] : undefined}
+          >
+            Awaiting PR
+          </th>
+          <th
+            onClick={() => resort("in-pr")}
+            className={sort.by === "in-pr" ? styles[sort.order] : undefined}
+          >
+            In PR
+          </th>
+          <th
+            onClick={() => resort("not-solvable")}
+            className={sort.by === "not-solvable" ? styles[sort.order] : undefined}
+          >
+            Not solvable
+          </th>
+          <th
+            onClick={() => resort("bot-error")}
+            className={sort.by === "bot-error" ? styles[sort.order] : undefined}
+          >
+            Bot error
+          </th>
+        </tr>
+      </thead>
+      <tbody className={collapsed ? styles.collapsed  : undefined}>
+        {rows.map((row) => {
+          const { progress } = row;
+          const href = `/status/migration/${row.name}`;
+          return (
+            <tr key={row.name}>
+              <td>
+                <a className="badge badge--secondary"
```

Comment:
Not sure if we want badges for these links. I liked it better when they were plain links.

Reply:
Got it. Let's leave it as is provided we can fix the hover/active states.